ddtrace/contrib/anthropic/__init__.py

```diff
@@ -87,10 +87,11 @@
 
 with require_modules(required_modules) as missing_modules:
     if not missing_modules:
-        from . import patch as _patch
+        # Required to allow users to import from `ddtrace.contrib.aiohttp.patch` directly
```

Comment:
We need to update this comment. Since we are copy-pasting this comment in a bunch of places we can make it more generic something like: Required to allow users to import from `ddtrace.contrib.integration_name.patch` directly
```suggestion
        # Required to allow users to import from `ddtrace.contrib.antropic.patch` directly
```